compiler/src/test-integration/java/io/neow3j/compiler/OracleContractTest.java

```diff
@@ -71,32 +69,39 @@ public void performRequest() throws Throwable {
         Await.waitUntilTransactionIsExecuted(response.getSendRawTransaction().getHash(), neow3j);
 
         // Invoke contract that requests data from oracle.
-        String url = "http://127.0.0.1:98239/test"; // Request URL that is not reachable
+        String url = "http://127.0.0.1:64928/test"; // Request URL that is not reachable
```

Comment:
sometimes this port can be available (open), and sometimes not? Or due to the wire mock it should be always unavailable?

If it's always unavailable, it's fine to leave like this. However, if it's not enforced by wire mock, then I suggest to add a random port therefor  every time the test runs.

Reply:
I added WireMock to actually mock the web service called by the oracle.
But I didn't manage to make it work fully. The request of the oracle to the web service times out.
It doesn't matter for this test, because we don't need to test the whole Oracle process just the `OracleContract`.